packages/pictograms/categories.yml

```
@@ -61,6 +61,10 @@ categories:
     - madrid--skyscrapers
     - madrid--statue
     - melbourne
+    - mexico--city--angel--of--independence
+    - mexico--city--museo--soumaya
+    - milan--duomo--di--milano
+    - milan--skyscrapers.svg
```

```suggestion
    - milan--skyscrapers
```

per @jnm2377 comments 👍